head.lisp

```diff
@@ -56,6 +68,18 @@
                  :test #'frames-overlap-p)
                 do (setf (head-number h) i)
                 collect h)))
+      (and (xlib:query-extension *display* "RANDR")
+             (loop
+                for i = 0 then (1+ i)
+                for h in
+                (delete-duplicates
```

Comment:
Consider factoring this (delete-duplicates ...) into (list-heads-xrandr) (or something similar